loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowParametersDeserializer.java

```diff
@@ -64,9 +64,10 @@ public LoadFlowParameters deserialize(JsonParser parser, DeserializationContext
                     parameters.setPhaseShifterRegulationOn(parser.readValueAs(Boolean.class));
                     break;
 
-                case "specificCompatibility":
+                case "specificCompatibility":   // Keep the old tag name for compatibility
+                case "t2wtSplitShuntAdmittance":
```

Comment:
Are you sure this is the proper way to implement that. We should base the compatibility assessment on the version (which is just keep for now, but has been added specially for this kind of modification).

Reply:
@MioRtia do you agree with this?

Reply:
@mathbagu I am okay with your suggestion. Just to be clear, it means that powsybl will be able to read previous versions but will always serialize JSON configuration files in the latest version, right?

Reply:
ok Miora. I will do what you propose, I mean, compatibility will be based on the attribute `version` of the `LoadFlowParameters` class. The new code will be able to read serializations recorded with previous versions (recorded with the tag `specificCompatibility` and `version 1.0`) and will always serialize new JSON configuration files in the new version using the tag `t2wtSplitShuntAdmittance` and `version 1.1`.

The next planned PR extends `splitShuntAdmittance` to the `threWindingsTransformers`. We prefer to use a specific configuration attribute for `twoWindingsTransformers` (`t2wtSplitShuntAdmittance`) and for `threeWindingsTransformers` (`t3wtSplitShuntAdmittance`). My initial idea was two separated PRs, but now I am reconsidering the possibility to include all changes in this PR (PR1198) to avoid increasing the version twice. What do you prefer?

Reply:
Even if another configuration attribute is added in another PR, it can still be in version 1.1 : we only increase version if a new version has been delivered in a release so you can keep two separate PRs. Next release will be in a bit more than a month, so there should be no need to increase the version for the other PR if it is merged in this timeframe.

Reply:
Done. The migration guide has also been updated.